src/form-datasets.rst

```diff
@@ -77,6 +79,59 @@ These files may also have any number of additional columns used in :ref:`choice
   ezeke_amasiri, Ezeke amasiri, afikpo_n
   poperi_amasiri, Poperi amasiri, afikpo_n
 
+.. _selects-from-geojson:
+
+Building selects from GeoJSON files
+------------------------------------
+
+*New in* `ODK Collect v2022.2.0 <https://github.com/getodk/collect/releases/tag/v2022.2.0>`_, `ODK Central v1.4.0 <https://forum.getodk.org/t/odk-central-v1-4/36886>`_
+
+.. warning::
+  GeoJSON attachments are not yet available in web forms (Enketo).
+
+
+GeoJSON files that follow `the GeoJSON spec <https://datatracker.ietf.org/doc/html/rfc7946>`_ can be used as datasets that populate select questions using ``select_one_from_file``. Selects from GeoJSON may be styled as maps using the :ref:`map appearance <select-from-map>` but can also use any other :ref:`select appearance <select-appearances>`. In order to be used by a form, a GeoJSON file:
+
+- *MUST* have a ``.geojson`` extension (NOT ``.json``)
+- *MUST* define a single top-level `FeatureCollection <https://datatracker.ietf.org/doc/html/rfc7946#section-3.3>`_
+- *MUST* include a unique identifier for each feature (by default, a top-level :th:`id`, falling back to an :th:`id` property, or can be :ref:`configured <customizing-label-and-value>`)
+- *MUST* only include features with `Point geometry <https://datatracker.ietf.org/doc/html/rfc7946#appendix-A.1>`_
+
+.. csv-table:: survey
+  :header: type, name, label
+
+  select_one_from_file museums.geojson,museum,Select the museum closest to you
+
+GeoJSON files referenced in forms can have any number of ``features`` and any number of custom ``properties``.
+
+.. code-block:: json
+
+    {
+      "type": "FeatureCollection",
+      "features": [
+          {
+              "type": "Feature",
+              "geometry": {
+                  "type": "Point",
+                  "coordinates": [
+                      7.0801379,
+                      46.5841618
+                  ]
+              },
+              "properties": {
+                  "id": "fs87b",
+                  "title": "HR Giger Museum",
+                  "annual_visits": 40000
+              }
+          }
+      ]
+    }
+
+Custom properties such as ``annual_visits`` above are displayed by :ref:`selects from map <select-from-map>`. All properties can be :ref:`referenced by any part of the form <referencing-values-in-datasets>`.
+
+A feature's :th:`geometry` can be accessed as ``geometry`` and is provided in :ref:`the ODK format <location-widgets>`. Given the GeoJSON file and the form definition above, if the user selected "HR Giger Museum", the value of ``${museum}`` would be ``"fs87b"``. The expression ``instance('museums')/root/item[id=${museum}]/geometry`` would evaluate to ``46.5841618 7.0801379 0 0`` which is a point in :ref:`the ODK format <location-widgets>`.
```

Comment:
A newline before The expression would make this more legible.

You don't need the second ODK format link.